dashboard/config/locales/en.yml

```diff
@@ -212,11 +212,16 @@ en:
     confirm:
       remove: 'Are you sure you want to leave this section?'
     remove: '(remove section)'
+    type:
+      picture: 'Picture Password'
+      word: 'Secret Words'
+      email: 'Email'
+      google_classroom: 'Google Classroom'
+      clever: 'Clever'
```

Comment:
https://youtu.be/TO5wryDdEI0?t=20 Every time I see "Clever" this is what I think of.